playbooks/capsule/vars/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+capsule_certs_tar: "/root/{{ ansible_nodename }}.tar.gz"
+capsule_server_group: "server-{{ inventory_hostname }}"
+capsule_server: "{{ groups[server_group][0] }}"
```

Comment:
Nitpick, but I found this variable name hard to understand.  It is the server for the capsule, so the katello host name.  But it's a little ambiguous in meaning to me.

Thinking about deploying a freeipa server or some other X type of server, it'd be great if this was just more generic and called "katello_server" so could just reuse the same vars.  Thoughts?